extensions/wikia/PortableInfobox/controllers/ApiQueryPortableInfobox.class.php

```diff
@@ -51,17 +53,24 @@ protected function runOnPageSet( ApiPageSet $pageSet ) {
 	 * @param $title
 	 * @return array
 	 */
-	private function sourceLabelsFallback( $infobox, $title ) {
+	private function metadataFallback( $infobox, $title ) {
 		global $wgCityId;
 
-		Wikia\Logger\WikiaLogger::instance()->info( 'Portable Infobox ApiQuery sourcelabels fallback' );
+		Wikia\Logger\WikiaLogger::instance()->info( 'Portable Infobox ApiQuery metadata fallback' );
 
 		$task = new Wikia\Tasks\Tasks\RefreshLinksForTitleTask();
+		$task->setTitle( $title );
 		$task->title( $title );
 		$task->call( 'refresh' );
 		$task->wikiId( $wgCityId );
 		$task->queue();
 
+		// TODO reparse and get fresh data
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)